podcasts/EpisodeArtwork.swift

```diff
@@ -45,12 +46,17 @@ class EpisodeArtwork {
     }
 
     private func loadEpisodeArtworkFromUrl(podcastUuid: String, episodeUuid: String) {
-        CacheServerHandler.shared.loadEpisodeArtworkUrl(podcastUuid: podcastUuid, episodeUuid: episodeUuid) { [weak self] imageUrl in
+        if !FeatureFlag.newShowNotesEndpoint.enabled && !FeatureFlag.episodeFeedArtwork.enabled {
```

Comment:
Maybe a guard statement here?

Reply:
Good one! Done in 610917957a91cbe7f56886d3602c00787c50783e